medusa/tv/series.py

```diff
@@ -2131,19 +2155,31 @@ def to_json(self, detailed=True):
             data['config']['release']['whitelist'] = bw_list.whitelist
 
         if detailed:
-            episodes = self.get_all_episodes()
             data['size'] = self.size
-            data['seasons'] = [list(v) for _, v in
-                               groupby([ep.to_json() for ep in episodes], lambda item: item['season'])]
+            data['showQueueStatus'] = self.show_queue_status
+            data['xemNumbering'] = [{'source': {'season': src[0], 'episode': src[1]},
+                                     'destination': {'season': dest[0], 'episode': dest[1]}}
+                                    for src, dest in viewitems(self.xem_numbering)]
+            data['sceneAbsoluteNumbering'] = self.scene_absolute_numbering
```

Comment:
@sharkykh this will result in a dynamic key: value mapping.
Like: [1: 1, 2: 2, 3: 4].
Thing is. Swagger doesn't like this. Or at least I haven't figured out how to configure this in swagger in a way that `https://editor.swagger.io/` accepts it as valid. So maybe we should make sure they all have a similar format. As in: [src: 1, dest: 1, src: 2: dest: 2, etc..] ?